src/modules/asab-config/ConfigContainers/ConfigContainer.js

```diff
@@ -13,60 +13,204 @@ import { useTranslation } from 'react-i18next';
 import { TreeViewComponent } from "./TreeViewComponent";
 import ConfigEditor from "./ConfigEditor";
 import ConfigList from "./ConfigList";
-// import reducer from './reducer';
+import ConfigImport from "./ConfigImport";
 
 function ConfigContainer(props) {
 
-	const { t, i18n } = useTranslation();
+	const App = props.app;
+	const ASABConfigAPI = App.axiosCreate('asab_config');
+	const serviceURL = App.getServiceURL('asab_config');
+	const { t } = useTranslation();
 
 	const configType = props.match.params.configType;
 	const configName = props.match.params.configName;
 
-	const [ createConfig, setCreateConfig ] = useState(false);
-
 	const homeScreenImg = props.app.Config.get('brand_image').full;
 	const homeScreenAlt = props.app.Config.get('title');
 
+	const [ treeData, setTreeData ] = useState({});
+	const [ createConfig, setCreateConfig ] = useState(false);
+	const [ chosenPanel, setChosenPanel ] = useState("configurator");
```

Comment:
@Aringoaway Can you write short comment what this state is here for?

```
const [ chosenPanel, setChosenPanel ] = useState("configurator"); // This is here because,.....
```

Reply:
Updated